include/flashinfer/group_gemm/wrapper.cuh

```diff
@@ -85,7 +85,7 @@ cudaError_t CutlassSegmentGEMMWrapper(CutlassSegmentGEMMHandler* handler, DType*
         cutlass::gemm::GemmShape<16, 8, 16>,     // Instruction Shape
         cutlass::epilogue::thread::LinearCombination<DType, 8, float, float>,  // Epilogue
         cutlass::gemm::threadblock::GemmBatchedIdentityThreadblockSwizzle,     // Swizzling Operator
-        8                                                                      // Stages
+        4                                                                      // Stages
```

Comment:
Adjust the size based on different shared memory instead of directly changing 8 to 4. cc @yzh119

Reply:
The value 4 was chosen based on the configuration in cutlass, which should be compatible with most scenarios. :smile:

Reply:
The example of the cutlass you provided is the SM80, not the SM90.